lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9159,6 +9159,8 @@ void ClangImporter::Implementation::swiftify(FuncDecl *MappedDecl) {
       returnHasLifetimeInfo = true;
     }
     for (auto [index, clangParam] : llvm::enumerate(ClangDecl->parameters())) {
+      if (index >= MappedDecl->getParameters()->size())
```

Comment:
While this protects against the crash, I don't think we want to apply `_SwiftifyImport` in _any_ case where there's a parameter mismatch, since they could all be off by 1. I would suggest changing it to an early exit before the loop:
```
if (ClangDecl->parameters()->size() != MappedDecl->getParameters()->size())
  return;
```

Reply:
Fixed!